wrapper/src/main/java/software/amazon/jdbc/HikariPooledConnectionProvider.java

```diff
@@ -156,7 +156,7 @@ public HikariPooledConnectionProvider(
   public boolean acceptsUrl(
       @NonNull String protocol, @NonNull HostSpec hostSpec, @NonNull Properties props) {
     final RdsUrlType urlType = rdsUtils.identifyRdsType(hostSpec.getHost());
-    return RdsUrlType.RDS_INSTANCE.equals(urlType);
+    return RdsUrlType.RDS_INSTANCE.equals(urlType) || hostSpec.getHost().contains(".proxied");
```

Comment:
I changed this temporarily so that the integrations can use the toxiproxy instance URLs and still use internal connection pools. We probably will have to come up with an alternative to this change.

Reply:
Should we add a new parameter to HikariPooledConnectionProvider constructor similar to HikariPoolMapping? A function/lambda that returns true or false depending on host name. It can check if pool can accept that host name, if host name is a RDS instance name.

Reply:
Added now